assets/prototype/domain/eventEditor/datetimes/dateForm/DateForm.tsx

```diff
@@ -19,8 +21,18 @@ const formatSecondaryField = (ticketPrice: number, toString = false): JSX.Elemen
 	return toString ? renderToString(<Currency quantity={ticketPrice} />, null) : <Currency quantity={ticketPrice} />;
 };
 
-const DateForm: React.FC<DateItemFormProps> = ({ id, formReset, relatedTickets, tickets = [], title }): JSX.Element => {
+const DateForm: React.FC<DateItemFormProps> = ({ id, formReset, title }): JSX.Element => {
 	const { description = '', name = '' } = useDatetimeItem({ id }) || {};
+	const { getRelations } = useRelations();
+	const tickets = useDatetimes();
+
+	const relatedTicketIds = id
+		? getRelations({
+				entity: 'datetimes',
+				entityId: id,
+				relation: 'tickets',
+		  })
+		: [];
```

Comment:
Is it possible to have this ternary stuff abstracted inside of `getRelations`?
Not a biggie, just asked.